build-system/eslint-rules/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "eslint-plugin-amphtml-internal",
+  "version": "0.0.0",
+  "private": true
```

Comment:
Consider adding a brief `"description"` for this package.

Reply:
Done.